tools/generator/tests/test_AnDOGenerator.py

```diff
@@ -83,7 +82,7 @@ def test_complex_parameters(self):
 
     def test_paths(self):
         ses = AnDOSession(self.expName, self.guid, self.sesID)
-
+        print(ses)
```

Comment:
I guess this is still a leftover from debugging? Shouldn't stay in the codebase.

Reply:
deleted